pkmn.py

```diff
@@ -11,7 +11,7 @@
 learning_rate = 1e-4
 gamma = 0.99 # discount factor for reward
 decay_rate = 0.99 # decay factor for RMSProp leaky sum of grad^2
-resume = False # resume from previous checkpoint?
+resume = True # resume from previous checkpoint?
```

Comment:
Won't work unless save.p is in the repo (the only currently available save file is "save_stable.p").

Reply:
This is true. The idea was that save.p gets overriden so often that it is impossible to keep my save.p the same as it is on GitHub. I don't feel strongly about this and can definitely rename save_stable.p to save.p